src/App.jsx

```diff
@@ -27,6 +28,7 @@ function App() {
           </Route>
         </Routes>
       </AuthProvider>
+      <RightSideDrawer />
```

Comment:
I think this should be inside the layouts , and that too in Header-> Appbar.
Please consider it and try to implement it there

Reply:
we will need it on login and all the other non layout pages also so , Please consider that too

Reply:
Resolved